disease/schemas.py

```diff
@@ -266,27 +266,24 @@ def schema_extra(schema: Dict[str, Any],
             }
 
 
-class MergedMatch(BaseModel):
-    """Represent merged concept in response to client."""
+class Extension(BaseModel):
+    """Value Object Descriptor Extension class."""
 
-    label: Optional[str]
-    concept_ids: List[str]
-    aliases: Optional[List[str]]
-    xrefs: Optional[List[str]]
-    pediatric_disease: Optional[bool]
+    type: str
+    name: str
+    value: Union[bool, List[str]]
 
-    class Config:
-        """Enables orm_mode"""
 
-        @staticmethod
-        def schema_extra(schema: Dict[str, Any],
-                         model: Type['MergedMatch']) -> None:
-            """Configure OpenAPI schema"""
-            if 'title' in schema.keys():
-                schema.pop('title', None)
-            for prop in schema.get('properties', {}).values():
-                prop.pop('title', None)
-            schema['example'] = {}  # TODO
+class ValueObjectDescriptor(BaseModel):
+    """VOD class."""
+
+    id: str
+    type: str
+    value: Any
```

Comment:
I think we shouldn't put `Any` here. I would consider creating a `Disease` class in `schemas.py` and have `value: Disease`.

Reply:
agreed!